api/guardrails.yaml

```
@@ -0,0 +1,703 @@
+          type: string
+          description: Configuration name
+        description:
+          type: string
+          nullable: true
+          description: Configuration description
+        config_yaml:
+          type: string
+          description: YAML content for config.yml
+        prompts_yaml:
+          type: string
+          description: YAML content for prompts.yml
+        colang:
+          type: string
+          description: Colang content
+        created_at:
+          type: string
+          format: date-time
+          description: Creation timestamp
+        updated_at:
+          type: string
+          format: date-time
+          description: Last update timestamp
+
+    VersionCreate:
+      type: object
+      required:
+        - name
+      properties:
+        name:
+          type: string
+          description: Version name
+          minLength: 1
+          maxLength: 255
+          example: v1.0.0
+        description:
+          type: string
+          description: Optional description
+          example: Initial release
+
+    VersionResponse:
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+          description: Version ID
+        config_id:
+          type: string
+          format: uuid
+          description: Parent configuration ID
+        name:
+          type: string
+          description: Version name
+        revision:
+          type: integer
+          description: Revision number
+          example: 1
+        is_active:
+          type: boolean
+          description: Whether this version is active
+          example: false
+        description:
+          type: string
+          nullable: true
+          description: Version description
+        created_at:
+          type: string
+          format: date-time
+          description: Creation timestamp
+
+    ActivateResponse:
+      type: object
+      properties:
+        status:
+          type: string
+          example: activated
+        version_id:
+          type: string
+          format: uuid
+        revision:
+          type: integer
+          example: 1
+
+    ReloadResponse:
+      type: object
+      properties:
+        status:
+          type: string
+          example: success
+        revision:
+          type: integer
+          example: 1
+        message:
+          type: string
+          example: Reloaded configuration revision 1
+
+    Error:
+      type: object
+      required:
+        - detail
+      properties:
+        detail:
+          type: string
+          description: Error message
+
+tags:
+  - name: Chat
+    description: Chat completion endpoints with guardrails
+  - name: Health
+    description: Health check and service information endpoints
+  - name: Configs
+    description: Guardrail configuration management
+  - name: Versions
+    description: Configuration version management
+  - name: Admin
+    description: Administrative operations
```

```suggestion
    description: Administrative operations

```
same for any other files